src/components/VerticalNav/VerticalNav.test.js

```diff
@@ -23,6 +23,12 @@ test('VerticalNav renders properly in mobile mode', () => {
   expect(component.render()).toMatchSnapshot();
 });
 
+test('VerticalNav renders properly with persistence off', () => {
```

Comment:
I don't feel like this test does much. To rest the persist ternary you could export WithPersist and NoPersist from VericalNav.js (not the index.js) and then import them in the test. You could then assert similar to below:
```jsx
const component = shallow(<VerticalNav persist />);
expect(component.find(WithPersist).exists()).toBe(true);
```
and
```jsx
const component = shallow(<VerticalNav persist={false} />);
expect(component.find(NoPersist).exists()).toBe(true);
```
This still doesn't cover everything, but it probably is not worth writing tests for the controlled HOC if it is going away.

Reply:
That makes sense. I mostly copypasted as a sanity check to make sure the component doesn't crash when rendered that way. I like your version better. You're full of little snippets for my code library @dmiller9911.